consensus/ibft/verifier.go

```diff
@@ -116,9 +118,12 @@ func (i *backendIBFT) IsValidProposalHash(proposal, hash []byte) bool {
 	return bytes.Equal(blockHash, hash)
 }
 
-func (i *backendIBFT) IsValidCommittedSeal(proposalHash, seal []byte) bool {
+func (i *backendIBFT) IsValidCommittedSeal(
+	proposalHash []byte,
+	committedSeal *messages.CommittedSeal,
+) bool {
 	//	seal was generated based on the proposal hash
-	validator, err := ecrecoverImpl(seal, wrapCommitHash(proposalHash))
+	validator, err := ecrecoverImpl(committedSeal.Signature, wrapCommitHash(proposalHash))
```

Comment:
Do we need an additional check here to make sure that `validator == committedSeal.Signer`?

Reply:
@zivkovicmilos I'm afraid I don't understand what you mean. The latest develop code check it. Do you think we can remove this check?

Reply:
Added at b44c2748eae722d4ffad0408f640f94127246649

Reply:
Added at b44c2748eae722d4ffad0408f640f94127246649